src/components/posts/PostForm.js

```diff
@@ -44,15 +56,38 @@ export const PostForm = () => {
               content: post.content
           })
           .then(() => history.push("/posts")) //This link string might be different for posts. Hasn't been coded yet.
-    }}
+    }   
+  }
+// debugger
+const urlId = postId.postId
+console.log('urlId: ', urlId);
+useEffect(() => {
+  getCategories()
+  .then(getPosts)
+  .then(posts => { 
+    const postbyId = posts.find(p => p.id === parseInt(urlId))
```

Comment:
This runs every time the form loads, so when adding a new post postById is undefined, breaking the code. I think a conditional should fix this.